app/extensions/string_extensions.rb

```diff
@@ -36,7 +36,7 @@ class String
 
   # :stopdoc:
   # Disable cop in order to make code more readable
-  # rubocop:disable Layout/AlignHash
+  # rubocop:disable Layout/HashAlignment,Layout/AlignHash
```

Comment:
I suggest simply
```ruby
rubocop:disable Layout/HashAlignment
```
RuboCop has recently made a bunch of changes to namespaces. I don't know when they changed this. I think it was a long time ago, and I simply missed it.

Reply:
That's what it was and it didn't work.  Both are needed I believe due to a mismatch between our current Rubocop version and the version used by CodeClimate.